src/features/broker-protection/comparison-functions.js

```diff
@@ -0,0 +1,449 @@
+import parseAddress from 'parse-address'
+
+/**
+ * @param userAge
+ * @param ageFound
+ * @return {boolean}
+ */
+export function isSameAge (userAge, ageFound) {
```

Comment:
@shakyShane I don't feel really strongly about this, but I would almost prefer each of these functions to be separate files in a utils folder, just from a code organization standpoint. They're pretty involved, so it would help to split up this monolith a bit.

Reply:
Done!